pghoard/restore.py

```diff
@@ -543,6 +511,32 @@ def single_download_progress(current_pos, expected_max,
         print("On systemd based systems you can run systemctl start postgresql")
         print("On SYSV Init based systems you can run /etc/init.d/postgresql start")
 
+    def process_one_chunk(self, *, transfer, basebackup_data_file, progress_callback, site, pgdata, tablespaces):
+        self.log.debug("Processing one chunk: %r", basebackup_data_file)
+        if isinstance(basebackup_data_file, tuple):
+            tmp_obj, tmp_metadata = basebackup_data_file
+            self.extract_one_backup(
+                obj=tmp_obj,
+                metadata=tmp_metadata,
+                pgdata=pgdata,
+                site=site,
+                tablespaces=tablespaces
+            )
+        else:
+            tmp, metadata = self.download_one_backup(
+                transfer=transfer,
+                basebackup_data_file=basebackup_data_file,
+                progress_callback=progress_callback,
+                site=site
+            )
+            self.extract_one_backup(
```

Comment:
Why not just move the `extract_one_backup` call outside the if/else block since it's identical in both branches. (and add a trailing comma to the argument list)